ldms/man/ldmsd_sampler_advertisement.man

```diff
@@ -173,13 +187,16 @@ aggregator (\fBagg\fR) runs on \fBnode-3\fR. All LDMSD listen on port 411.
 
 The sampler daemons collect the \fBmeminfo\fR set, and they are configured to
 advertise themselves and connect to the aggregator using sock on host
-\fBnode-3\fR at port 411. The following are the configuration files of the
-\fBsamplerd-1\fR and \fBsamplerd-2\fR.
+\fBnode-3\fR at port 411. They will try to reconnect to the aggregator every 10
+seconds until the connection is established. The following are the configuration
+files of the \fBsamplerd-1\fR and \fBsamplerd-2\fR.
 
 .EX
 .B
 > cat samplerd-1.conf
 .RS 4
+# Create a listening endpoint
+listen xprt=sock port=411
```

Comment:
This isn't strictly needed for the advertiser function, correct? What is the reasoning for including it in the example?